Good-Bids/goodbids
# [#698] Pending with Skipped Payments

# Summary

- Cleans up unused api calls
- Adds check for skipped payment step

## Issues

- #698

## Testing Instructions

1. Create a new site and follow onboarding up to payment set up
2. Skip payment set up
3. View Set Up Guide and see that launching is disabled.

## Screenshots

![image](https://github.com/Good-Bids/goodbids/assets/48295174/848f2196-0a47-4ba2-885e-76c1a2f38741)
